bin/sce_qc_report.R

```diff
@@ -121,6 +122,7 @@ metadata_list <- list(
   unfiltered_cells = ncol(unfiltered_sce),
   total_reads = sce_meta$total_reads,
   mapped_reads = sce_meta$mapped_reads,
+  filtering_method = filtered_sce_meta$filtering_method,
```

Comment:
I don't think the order really matters (json objects are unordered, officially, so anybody reading it should not expect a particular order), but I would probably go with putting the method right after the filtered/unfiltered set.
```suggestion
  filtering_method = filtered_sce_meta$filtering_method,
  total_reads = sce_meta$total_reads,
  mapped_reads = sce_meta$mapped_reads,
```